tests/product/resources/connector_permissions_warning.txt

```diff
@@ -1,24 +1,28 @@
 
-Warning: [master] Error validating /etc/opt/prestoadmin/connectors/tpch.properties
+Fatal error: [%(slave3)s] Error validating /etc/opt/prestoadmin/connectors/tpch.properties
```

Comment:
Any particular reason you have host names enclosed within %s or this can be avoided?